_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,137 @@
+
+If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+
+An empty error report in certain kinds of crashes is not empty anymore, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code! He also fixed a crash when NewPipe encounters an empty comment. Now it will be displayed just as empty as it is.
+
+Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab (and potentially other locations), whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
+
+The license pop-up doesn't disappear when you change orientation now. This was introduced by [@nmurali94](https://github.com/nmurali94). And [@budde92](https://github.com/budde25) ensured that you can now delete a file you just downloaded and then try to re-download it without the app crashing. We are not sure why you'd want to do this, but now you can do so without any worries.
+
+[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay for less redundancy. When auto-queue is enabled, Newpipe wants to add a stream to the queue the moment you open a video. But if there are no video suggestions on that page, it causes a crash. This has been fixed by [@wb9688](https://github.com/wb9688).
+
+## Development
+
+Development. Where all the magic which keeps this app running happens. It saw some improvements in this release: (Warning: Nerd talk ahead)
+
+- The extractor will ignore OTF streams now, since they don't work with progressive HTTP, and the extractor can't work with DASH just yet. This has been a major point of inconvenience for a lot of users (and us, too!) so we're eager to mitigate the problem until the proper fix sometime later. ([@wb9688](https://github.com/wb9688))
+
+- A Checkstyle rule was added so it would show an error when a local variable which could be `final` wasn't declared as such. ([@XiangRongLin](https://github.com/XiangRongLin))
+
+- A library we use, PrettyTime, has been updated to a newer version so that a workaround can be removed from NewPipe's code. We get a perfomance improvement as a bonus! ([@B0pol](https://github.com/B0pol)
+
+- Numbers and uppercase letters are now allowed in the application ID. ([@Stypox](https://github.com/Stypox))
+
+- Contributors need to to discuss their contributions now prior to opening a PR. ([@gkeegan](https://github.com/gkeegan))
+	
+- Some deprecations as well as a lot of code cleanup happened. ([@TacoTheDank](https://github.com/TacoTheDank))
+
+- The code now prevents the YouTube website from accessing the available Java packages from NewPipe's parser. This removes a possible attack angle. ([@wb9688](https://github.com/wb9688))
```

Comment:
It's the correct term, so we should use it.

```suggestion
- The code now prevents the YouTube website from accessing the available Java packages from NewPipe's parser. This removes a possible attack surface. ([@wb9688](https://github.com/wb9688))
```

Reply:
still think angle sounds better :(

Reply:
Use official terms. We can't suddenly say "Pull suggestion" instead of "Pull request", can we? O_O